src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -20,6 +20,9 @@ public CommandResult(String feedbackToUser) {
         this.feedbackToUser = feedbackToUser;
         relevantPersons = null;
     }
+    public String GetCommandResult() {
```

Comment:
method signature should be in camel notation as per java convention and have it as a getter method.